pkg/bosh/manifest/dns.go

```diff
@@ -207,12 +237,43 @@ func (dns *boshDomainNameService) Reconcile(ctx context.Context, namespace strin
 	return nil
 }
 
+func (dns *boshDomainNameService) createCorefile(namespace string) string {
+	rewrites := make([]string, 0)
+	for _, alias := range dns.Aliases {
+		for _, target := range alias.Targets {
+			from := alias.Domain
+			if target.Query == "_" {
+				from = strings.Replace(from, "_", target.InstanceGroup, 1)
```

Comment:
This could use a comment (because the replacement here is weird and BOSH-y)

Reply:
Fixed in https://github.com/cloudfoundry-incubator/cf-operator/pull/642/commits/1c14f1e9d52238e87e789958171835ef0757de8b.